Packs/Anomali_Enterprise/Integrations/Anomali_Enterprise/Anomali_Enterprise.py

```diff
@@ -0,0 +1,290 @@
+        'job_id': response.get('jobid', '')
+    }
+
+    return CommandResults(
+        outputs_prefix='AnomaliEnterprise.ForensicSearch',
+        outputs_key_field='job_id',
+        outputs=start_search_outputs,
+        readable_output=tableToMarkdown(name="Forensic search started:", t=start_search_outputs, removeNull=True),
+        raw_response=response
+    )
+
+
+def get_search_job_result(client: Client, args: Dict) -> CommandResults:
+    """Get the search job result.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+
+    Returns:
+        CommandResults.
+    """
+    job_id = str(args.get('job_id'))
+
+    response = client.get_search_job_result_request(job_id)
+    if 'error' in response:
+        raise Exception(f"{str(response.get('error'))}. Job ID might have expired.")
+
+    response.update({'job_id': job_id})
+    if not response.get('complete'):
+        human_readable = f'job ID: {job_id} is still in progress.'
+        response.update({'status': 'in progress'})
+    else:
+        if response.get('totalMatches'):
+            human_readable = tableToMarkdown(name="Forensic search results:", t=response.get('streamResults'),
+                                             removeNull=True)
+        else:
+            human_readable = f'No matches found for the given job ID: {job_id}.'
+            response.update({'status': 'completed'})
+
+    return CommandResults(
+        outputs_prefix='AnomaliEnterprise.ForensicSearch',
+        outputs_key_field='job_id',
+        outputs=response,
+        readable_output=human_readable,
+        raw_response=response
+    )
+
+
+def dga_domain_status(client: Client, args: dict) -> CommandResults:
+    """Search domain DGA status.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+
+    Returns:
+        CommandResults.
+    """
+    domains = argToList(str(args.get('domains')))
+
+    response = client.domain_request(domains)
+
+    domains_data = response.get('data', {})
+    outputs = []
+    for domain in domains:
+        output = {
+            'domain': domain,
+            'malware_family': domains_data.get(domain, {}).get('malware_family'),
+            'probability': domains_data.get(domain, {}).get('probability')
+        }
+        outputs.append(output)
+    return CommandResults(
+        outputs_prefix='AnomaliEnterprise.DGA',
+        outputs_key_field='domain',
+        outputs=outputs,
+        readable_output=tableToMarkdown(name="Domains DGA:", t=outputs, removeNull=True),
+        raw_response=response
+    )
+
+
+def domain_command(client: Client, args: dict) -> CommandResults:
+    """Search domain DGA status.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+
+    Returns:
+        CommandResults and DBotScore.
+    """
+    domain = str(args.get('domain'))
+
+    response = client.domain_request([domain])
```

Comment:
Please use `argToList` for `isArray` support. Following that, please wrap the code below with a for loop, so each domain will have its own `CommandResult`. Please note, multiple `CommandResults` should be supported following the merge of this PR: https://github.com/demisto/content/pull/9613

Reply:
This is not supporting an array. The API needs to get the IOCs in a list.

Reply:
Indeed, the code doesn't support an array, but the yaml specifies the `domain` as `isarray: true` - if the API can natively support domain list, then we better take advantage of it

Reply:
Agreed, handled in the code, pending your PR.